tests/pytorch/test_models_detection_pt.py

```diff
@@ -14,19 +14,26 @@
 
 
 @pytest.mark.parametrize(
-    "arch_name, input_shape, output_size, out_prob",
+    "arch_name, input_shape, output_size, out_prob, train_mode",
     [
-        ["db_resnet34", (3, 512, 512), (1, 512, 512), True],
-        ["db_resnet50", (3, 512, 512), (1, 512, 512), True],
-        ["db_mobilenet_v3_large", (3, 512, 512), (1, 512, 512), True],
-        ["linknet_resnet18", (3, 512, 512), (1, 512, 512), False],
-        ["linknet_resnet34", (3, 512, 512), (1, 512, 512), False],
-        ["linknet_resnet50", (3, 512, 512), (1, 512, 512), False],
+        ["db_resnet34", (3, 512, 512), (1, 512, 512), True, True],
+        ["db_resnet34", (3, 512, 512), (1, 512, 512), True, False],
+        ["db_resnet50", (3, 512, 512), (1, 512, 512), True, True],
+        ["db_resnet50", (3, 512, 512), (1, 512, 512), True, False],
+        ["db_mobilenet_v3_large", (3, 512, 512), (1, 512, 512), True, True],
+        ["db_mobilenet_v3_large", (3, 512, 512), (1, 512, 512), True, False],
+        ["linknet_resnet18", (3, 512, 512), (1, 512, 512), True, True],
+        ["linknet_resnet18", (3, 512, 512), (1, 512, 512), True, False],
+        ["linknet_resnet34", (3, 512, 512), (1, 512, 512), True, True],
+        ["linknet_resnet34", (3, 512, 512), (1, 512, 512), True, False],
+        ["linknet_resnet50", (3, 512, 512), (1, 512, 512), True, True],
+        ["linknet_resnet50", (3, 512, 512), (1, 512, 512), True, False],
```

Comment:
I just discovered we can stack parametrize parameter [from documentation](https://docs.pytest.org/en/7.2.x/how-to/parametrize.html) (quite useful for `train_mode` argument):
```python
import pytest


@pytest.mark.parametrize("x", [0, 1])
@pytest.mark.parametrize("y", [2, 3])
def test_foo(x, y):
    pass
```

We can keep it like that for this PR, that's fine :+1:

Reply:
Ah nice good to know :+1: 
Yes, I also think that we can slim it down in another PR  :)